compiler/src/dotty/tools/io/Path.scala

```diff
@@ -160,8 +158,17 @@ class Path private[io] (val jpath: JPath) {
     val p = parent
     if (p isSame this) Nil else p :: p.parents
   }
+
+  def ext: FileExtension = Path.fileExtension(name)
+
   // if name ends with an extension (e.g. "foo.jpg") returns the extension ("jpg"), otherwise ""
-  def extension: String = Path.extension(name)
+  def extension: String = ext.toLowerCase
+
+  def hasExtension(ext: FileExtension, exts: FileExtension*): Boolean = {
```

Comment:
It seems this method is only called with exactly one argument. Remove the `exts` parameter?